disable_odoo_online/models/publisher_warranty_contract.py

```diff
@@ -0,0 +1,15 @@
+# Copyright (C) 2013 Therp BV (<http://therp.nl>).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, models
+from odoo.release import version_info
+
+
+class PublisherWarrantyContract(models.AbstractModel):
+    _inherit = 'publisher_warranty.contract'
+
+    @api.multi
+    def update_notification(self, cron_mode=True):
+        if version_info[5] == 'e':
```

Comment:
PublisherWarrantyContract

Reply:
Can you explain more?

Reply:
I honestly don't remember. Let's just assume I was being dumb!